assets/lambda/lib/eventHandler.ts

```diff
@@ -0,0 +1,92 @@
+const logClass = require('lambda-log');
+import * as eb from '@aws-sdk/client-eventbridge';
+import {APIGatewayProxyEvent} from 'aws-lambda';
+
+const {SYSTEM_EVENT_BUS_NAME, MTFX_AWS_REGION} = process.env;
+
+const ebClient = new eb.EventBridgeClient({region: MTFX_AWS_REGION});
+
+export interface IProxyEvent<T> {
```

Comment:
in other parts of code intefaces are declared without `I` prefix - let keep one style.
```suggestion
export interface ProxyEvent<T> {
```